contracts/utils.ts

```diff
@@ -29,7 +29,8 @@ export const networks = {
 export function handlerForNetwork(networkKey: string, contractKey: string) {
   try {
     const network = networks[networkKey];
```

Comment:
These 3 lines are exactly the same as in `httpProviderForNetwork`, may we reuse them or delete one of these methods?

Reply:
I agree, let's reuse `httpProviderForNetwork` here to avoid duplication.

Reply:
Done